src/qt/transactiontablemodel.cpp

```diff
@@ -520,8 +514,6 @@ QVariant TransactionTableModel::txStatusDecoration(const TransactionRecord *wtx)
     case TransactionStatus::OpenUntilBlock:
     case TransactionStatus::OpenUntilDate:
         return GUIUtil::getThemedQColor(GUIUtil::ThemedColor::TX_STATUS_OPENUNTILDATE);
-    case TransactionStatus::Offline:
```

Comment:
Should drop `TX_STATUS_OFFLINE`

Reply:
https://github.com/dashpay/dash/pull/3694/commits/c506b1113e0127f26748fbfd7b721170032019e0

Reply:
c506b11 isn't enough, should also drop https://github.com/dashpay/dash/blob/develop/src/qt/guiutil.cpp#L156 and https://github.com/dashpay/dash/blob/develop/src/qt/guiutil.cpp#L172

Reply:
git grep is useful... I should use it. https://github.com/dashpay/dash/pull/3694/commits/d8f8592077ab05abdbfeb8f4a2f61b66bcbc6f0d